internal/controlplane/handlers_evalstatus.go

```diff
@@ -126,7 +126,10 @@ func sortEntitiesEvaluationStatus(
 				profileStatuses[p.Profile.ID] = buildProfileStatus(&p, profileStatusList)
 			}
 
-			stat := buildRuleEvaluationStatusFromDBEvaluation(ctx, &p, e)
+			stat, err := buildRuleEvaluationStatusFromDBEvaluation(ctx, &p, e)
+			if err != nil {
+				return nil, nil, nil, fmt.Errorf("building rule evaluation status: %w", err)
+			}
```

Comment:
Do we want to have an error generating a single remediation URL cause the entire status list to fail?

I suspect we want to log and swallow the error in `buildRuleEvaluationStatusFromDBEvaluation`, so that the listing degrades gracefully in the face of malformed data / errors.